src/tests/JIT/Regression/JitBlue/Runtime_71592/test71592.cs

```diff
@@ -27,7 +27,7 @@ public class Test
         var t1 = new Test () { foo = 1 };
         var t2 = new Test () { foo = 2 };
         if (t1 == t2)
-            return 0;
+            return 100;
         return 100;
```

Comment:
```suggestion
        return 101;
```
when it should fail?

Reply:
This was supposed to be a test for an infinite loop in an optimization pass, so
the == call is only there to make sure the problematic code is called.